lib/config/presets/index.spec.ts

```diff
@@ -307,6 +311,41 @@ describe('config/presets/index', () => {
         labels: ['self-hosted resolved'],
       });
     });
+
+    it('gets preset value from cache when it has been seen', async () => {
+      config.extends = ['github>username/preset-repo'];
+      config.packageRules = [
+        {
+          matchManagers: ['github-actions'],
+          groupName: 'github-actions dependencies',
+        },
+      ];
+      gitHub.getPreset.mockResolvedValue({
```

Comment:
```suggestion
      gitHub.getPreset.mockResolvedValueOnce({
```

We want to make sure it only resoved once